src/XMakeBuildEngine/Logging/BaseConsoleLogger.cs

```diff
@@ -370,7 +370,7 @@ internal bool IsVerbosityAtLeast(LoggerVerbosity checkVerbosity)
         /// <summary>
         /// Sets foreground color to color specified
         /// </summary>
-        internal static void SetColor(ConsoleColor c)
+        public static void SetColor(ConsoleColor c)
```

Comment:
I had to change it to public to eliminate the code duplication. You had a copy paste of the function in Logging/ConsoleLogger.cs